constants/index.js

```diff
@@ -0,0 +1,10 @@
+const DeliveryStatus = {
```

Comment:
Initially I tried to create this file in the `models` directory, but that doesn't work with the way we're dynamically initializing the sequelize models. I went with the easy fix of sticking the file in a new directory. Lmk if you have any differing thoughts on how to make this accessible outside the `PatientDelivery` model.

Reply:
I think this was the right choice... I refactored it a bit to pull the constants out to a separate file in case we have others to add...